apps/pipelines/nodes/helpers.py

```diff
@@ -65,3 +65,28 @@ def set(self, data):
         participant_data.save(update_fields=["data"])
 
         self.session.participant.update_name_from_data(data)
+
+    def get_schedules(self):
+        """
+        Returns all active scheduled messages for the participant in the current experiment session.
+        """
+        from apps.events.models import ScheduledMessage
+
+        experiment = self.session.experiment
+        participant = self.session.participant
+        team = self.session.experiment.team
+        messages = (
+            ScheduledMessage.objects.filter(
+                experiment=experiment,
```

Comment:
Nit: To save a few milliseconds here, you can use `self.session.experiment_id` and `self.session.participant_id` instead. That way we don't have to fire two queries to load the instances